### ISSUE TITLE
Error on torch.jit.load the Pancreas CT Seg bundle 

### ISSUE THREAD

**Issue by MMelQin** [Project-MONAI/model-zoo#50]
**Describe the bug**
Tried to use the [Pancreas seg bundle](https://github.com/Project-MONAI/model-zoo/tree/dev/models/pancreas_ct_dints_segmentation) in an application built with MONAI Deploy App SDK and its bundle inference operator, but ran into error on loading the TorchScript, whereas the Spleen CT Seg bundle worked perfectly.

To isolate the problem, ran Python3 and tried to load TorchScript but saw the same error.  

**To Reproduce**
1. Ensure the correct PyTorch version is installed, >=1.10.0
```
mqin@mingq-dt:~/src/monai-app-sdk$ pip3 list | grep torch
pytorch-ignite                0.4.5
torch                         1.10.2
```
2. Go to model zoo folder for [Pancreas Bundle ](https://github.com/Project-MONAI/model-zoo/blob/dev/models/pancreas_ct_dints_segmentation/large_files.yml)and find the model.ts download URL
3. I used gdown to download the model.ts, instead of MONAI Bundle utility
```
gdown "https://drive.google.com/uc?export=download&id=15xvegu_sRFuDNK3U28PGcqqbd6P89ScI"
```
4. Run python3 interactively
```
mqin@mingq-dt:~/src/monai-app-sdk$ python3
Python 3.7.5 (default, Feb 23 2021, 13:22:40) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
>>> device
device(type='cuda')
>>> torch.jit.load("/tmp/model_multi_ai/pancrea_ct_dints/model.ts", map_location=device)
```
5. See the Traceback
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mqin/.local/lib/python3.7/site-packages/torch/jit/_serialization.py", line 161, in load
    cpp_module = torch._C.import_ir_module(cu, str(f), map_location, _extra_files)
RuntimeError: 

prim::RaiseException(str msg) -> ():
Expected at most 1 arguments but found 2 positional arguments.
:
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/instancenorm.py", line 379
    def _check_input_dim(self, input):
        if input.dim() not in (4, 5):
            raise ValueError('expected 4D or 5D input (got {}D input)'
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                             .format(input.dim()))
                             ~~~~~~~~~~~~~~~~~ <--- HERE
Serialized   File "code/__torch__/torch/nn/modules/instancenorm.py", line 34
    if _7:
      _8 = torch.format(_5, torch.dim(input))
      ops.prim.RaiseException(_8, "builtins.ValueError")
      ~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    else:
      pass
'InstanceNorm3d._check_input_dim' is being compiled since it was called from 'InstanceNorm3d.forward'
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/instancenorm.py", line 67
    def forward(self, input: Tensor) -> Tensor:
        self._check_input_dim(input)
        ~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    
        if input.dim() == self._get_no_batch_dim():
Serialized   File "code/__torch__/torch/nn/modules/instancenorm.py", line 18
  def forward(self: __torch__.torch.nn.modules.instancenorm.InstanceNorm3d,
    input: Tensor) -> Tensor:
    _0 = (self)._check_input_dim(input, )
                                 ~~~~~ <--- HERE
    _1 = torch.eq(torch.dim(input), (self)._get_no_batch_dim())
    if _1:
```

**Expected behavior**
Expected the bundle/TorchScript loaded without any errors, same as the Spleen CT.

**Screenshots**
See above for the console output

**Environment**

Ensuring you use the relevant python executable, please paste the output of:

```
python -c 'import monai; monai.config.print_debug_info()'
```
The TS failed to load with `torch.jit.load()` on consuming the TS as plain vanilla TS (even though the TS as a bundle had been generated with monai>=0.9.0) 

**Additional context**
I was building a multi-ai app with the App SDK, and was just trying out using multiple bundles from the Model Zoo. Execution was fine for the Spleen CT bundle, but Pancreas was giving the report error.

The BRATS_MRI bundle TS file failed, too, when jit.load.
```
>>> torch.jit.load("/tmp/model_multi_ai/brats_mri/model.ts", map_location=device)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mqin/.local/lib/python3.7/site-packages/torch/jit/_serialization.py", line 161, in load
    cpp_module = torch._C.import_ir_module(cu, str(f), map_location, _extra_files)
RuntimeError: 

prim::RaiseException(str msg) -> ():
Expected at most 1 arguments but found 2 positional arguments.
:
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/functional.py", line 2352
        size_prods *= size[i + 2]
    if size_prods == 1:
        raise ValueError("Expected more than 1 value per channel when training, got input size {}".format(size))
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
Serialized   File "code/__torch__/torch/nn/functional.py", line 489
    size_prods0 = size_prods1
  if torch.eq(size_prods0, 1):
    ops.prim.RaiseException(torch.format(_154, size), "builtins.ValueError")
    ~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
  else:
    pass
'_verify_batch_size' is being compiled since it was called from 'group_norm'
Serialized   File "code/__torch__/torch/nn/functional.py", line 13
    bias: Optional[Tensor]=None,
    eps: float=1.0000000000000001e-05) -> Tensor:
  _0 = __torch__.torch.nn.functional._verify_batch_size
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
  _1 = torch.mul(torch.size(input, 0), torch.size(input, 1))
  _2 = [torch.floordiv(_1, num_groups), num_groups]
'group_norm' is being compiled since it was called from 'GroupNorm.forward'
Serialized   File "code/__torch__/torch/nn/modules/normalization.py", line 14
  def forward(self: __torch__.torch.nn.modules.normalization.GroupNorm,
    input: Tensor) -> Tensor:
    _0 = __torch__.torch.nn.functional.group_norm
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    weight = self.weight
    bias = self.bias

>>>
```

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

Could you please help take a look at this issue?
Maybe something is wrong with the PyTorch version when exporting to TorchScript model?

Thanks in advance.

**Comment by yiheng-wang-nv** [comment]
Hi @dongyang0122 , could you please help to double confirm the pytorch version that is used for this bundle? I tried with pytorch 1.12.0 and it is loaded successfully, but 1.10.0 failed (same as @MMelQin found). Thanks!

**Comment by MMelQin** [comment]
Once `torch==1.12.0` is installed and used in the app, jit.load succeeded. This issue can be closed per PR #54
